samples/snippets/src/main/java/com/example/spanner/CreateBackupWithEncryptionKey.java

```diff
@@ -110,7 +102,7 @@ static Void createBackupWithEncryptionKey(DatabaseAdminClient adminClient,
             backup.getProto().getCreateTime().getSeconds(),
             backup.getProto().getCreateTime().getNanos(),
             OffsetDateTime.now().getOffset()),
-        ((CustomerManagedEncryption) backup.getEncryptionConfig()).getKmsKeyName()
+        backup.getEncryptionInfo().getKmsKeyVersion()
```

Comment:
According to [this](https://github.com/GoogleCloudPlatform/golang-samples/pull/2001#discussion_r596531313), this field will not necessarily be updated right away, and could return null, so it's probably not a good idea to use it here.

Reply:
Changed this to be the user supplied `keyName` instead.